doc/Rules.md

```diff
@@ -1012,7 +1012,7 @@ Parameters: style ("one", "ordered", "one_or_ordered", "zero"; default "one_or_o
 This rule is triggered for ordered lists that do not either start with '1.' or
 do not have a prefix that increases in numerical order (depending on the
 configured style). The less-common patterns of using '0.' as a first prefix or
-for all prefixes is also supported.
+for all prefixes are also supported.
```

Comment:
The appropriate correction in this sentence would be to change "patterns" to "pattern", rather than to change the verb.